client/src/components/books/Books.jsx

```diff
@@ -0,0 +1,94 @@
+import React from 'react';
```

Comment:
Not sure if you're still using this component, looks like you just hardcoded some books here. If you're not using it, then take it out, if you are then why the hardcoding?

Reply:
oh yes. I am not using this. I will take it out. Thank you.